tests/observability/metrics/conftest.py

```
@@ -389,13 +389,13 @@ def initiate_metric_value(request, prometheus):
 
 
 @pytest.fixture()
-def vm_for_vm_disk_allocation_size_test(namespace, unprivileged_client, golden_images_namespace):
+def vm_for_vm_disk_allocation_size_test(namespace, admin_client, unprivileged_client, golden_images_namespace):
```

Ruff (0.14.5)

Unused function argument: `unprivileged_client`

(ARG001)